src/EnvChecker.php

```diff
@@ -63,4 +61,42 @@ protected function checkUndocumented(array $meta): array
             $meta
         );
     }
+
+    protected function complain(array $errors): void
+    {
+        if (0 === sizeof($errors)) {
+            return;
+        }
+
+        $this->registerExceptionHandler();
+
+        throw new InvalidEnvException(implode('; ', $errors));
+    }
+
+    protected function registerExceptionHandler()
+    {
+        if ('cli' === php_sapi_name() || ini_get('display_errors')) {
+            return;
+        }
+
+        $originalExceptionHandler = set_exception_handler(
+            static function (\Throwable $e) {
+                if (!$e instanceof InvalidEnvException) {
+                    return;
+                }
```

Comment:
We're only interested in outputting to the browser when there was an InvalidEnvException.